src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -25,27 +26,30 @@
 import io.cucumber.java.en.Then;
 
 public class ConditionsSteps extends Steps {
-	
-	private ConditionsPage conditionsPage;
```

Comment:
Seems you are changing the format from the former formatting , was it a recommendation ?

Reply:
Yes it was a recommendation to have the test create its own patient